src/paper/step1_comparison_inv0_1.py

```diff
@@ -28,8 +28,8 @@
 #%%
 
 # I use the current working directory of the file to store the folder with the data for ease (FIR_LP_filter/).
-FOLDER_FOR_DATA = Path.cwd()/'measurements_12_05_22'
-if not FOLDER_FOR_DATA.exists():   
+FOLDER_FOR_DATA = Path('/mnt/data_folder')/'measurements_12_05_22/new_record_prop_channel/'
```

Comment:
Niko, auta mou kanoun break to diko mou installation.

Reply:
Στο αρχείο decimation_comparison.py στον φάκελο paper το έχω διαφορετικά. Αυτό τρέχει σωστά?